### DISCUSSION TITLE
CMYK -> SRGB conversion results in color change

### DISCUSSION THREAD

**Discussion by diz000** [libvips/libvips#3968] [General]
## Questions, enhancements, tips, etc.
I have a JPEG image with CMYK colour space. When I try to convert to srgb I get an image with slightly changed colors.
Not sure if it's a bug. Probably similar to #3159 . Didn't find a solution there. Would be great if someone could help solving this issue.

**To Reproduce**
Use command
`vips colourspace input.jpg output.jpg srgb `

**Screenshots**
Input:
![input](https://github.com/libvips/libvips/assets/54627994/42a5314e-5e50-42f0-90c5-6361b54bf6d7)

Output:
![output](https://github.com/libvips/libvips/assets/54627994/bde24aef-9f66-434b-ab99-8208fe3771c9)

**Environment**
 - OS: Ubuntu 20.04
 - Vips: 8.15.1

**Comment by lovell** [comment]
Given the use of ICC profiles, you able to try `vips icc_transform input.jpg output.jpg srgb` (as discussed in the https://github.com/libvips/libvips/issues/3159 issue you linked to)?

**Reply by 110111011** [reply]
Similar issue, but probably unrelated. Generates the same result (as expected).

**Comment by jcupitt** [comment]
I think I found it -- the `input.jpg` image has no colour profile (it's a raw cmyk image), so whatever image display or conversion program you use will pick a fallback for you. Different programs will have different fallback profiles, so you'll get varying results.

You can check for a profile with eg.:

```
$ vipsheader -a cmyk7.jpg  | grep icc
$ 
```

**Comment by diz000** [comment]
Thank you for your answers. Seems like @jcupitt is right. Many thanks!
